projects/dotcms/src/lib/api/DotApiContent.ts

```diff
@@ -1,5 +1,22 @@
 import { DotCMSHttpClient } from '../utils/DotCMSHttpClient';
-import { DotCMSContent, DotCMSError } from '../models';
+import { DotCMSContent, DotCMSContentQuery, DotCMSError } from '../models';
+
+function populateQueryUrl(params: DotCMSContentQuery): string {
+    let url = '';
+
+    const attrs = {
+        depth: `/depth/${params.depth}`,
+        limit: `/limit/${params.limit}`,
+        offset: `/offset/${params.offset}`,
+        orderBy: `/orderby/${params.orderBy}`
+    };
+
+    for (const key of Object.keys(params)) {
+        url += key !== 'query' ? attrs[key] : '';
+    }
```

Comment:
I thin is better do this like:

`${url}/depth/${params.depth}/limit/${params.limit}/offset/${params.offset}/orderby/${params.orderBy}`

Reply:
if not all params are received, then an Url with not needed params will be sent:
`${url}/depth/limit/offset/orderby/`

Reply:
But if a params is not receive,  an Url with not needed params will be sent with this code